src/test/coll.rs

```diff
@@ -720,3 +721,77 @@ async fn find_one_and_delete_hint_server_version() {
         assert!(res.is_ok());
     }
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
```

Comment:
this change looks like it's from @utkmehta 's changes, I think the rebase might not have happened properly if it's being included in the changes here.

Reply:
I actually wrote this test, it's included as a prose test in the crud spec https://github.com/mongodb/specifications/tree/master/source/crud/tests#errinfo-is-propagated

Reply:
Oh okay, well then I think we can just remove this one then. It's almost identical to the [one already implemented](https://github.com/mongodb/mongo-rust-driver/blob/master/src/test/coll.rs#L31).